src/redux/users/User.type.ts

```diff
@@ -0,0 +1,31 @@
+import { ADD_USER, SET_USERS } from './actionTypes';
```

Comment:
Using `type.ts` is absolutely alright can you also look into modules 
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html